presto-spark-base/src/main/java/com/facebook/presto/spark/RddAndMore.java

```diff
@@ -33,15 +39,24 @@
 {
     private final JavaPairRDD<MutablePartitionId, T> rdd;
     private final List<PrestoSparkBroadcastDependency<?>> broadcastDependencies;
-
+    private final Optional<PartitioningHandle> partitioningHandle;
```

Comment:
is this just to know if it's broadcast? Mabye we can just have a boolean?

Reply:
You are right, we are just using it for broadcast check & presence of partitioning handle. We may need it to determine other partitioning types in future